packages/Webkul/Product/src/Models/Product.php

```diff
@@ -321,7 +321,7 @@ public function getEditableAttributes($group = null, $skipSuperAttribute = true)
     public function getAttribute($key)
     {
         if (! method_exists(static::class, $key)
-            && ! in_array($key, ['parent_id', 'attribute_family_id'])
+            && ! in_array($key, ['pivot', 'parent_id', 'attribute_family_id'])
```

Comment:
Hi, may I know the reason because I haven't seen any `pivot` column in the `products` table?

Reply:
Hi @devansh-webkul 
It's coming in the case of linked products (i.e. up_sells, cross_sells, and related_products)

![error-related-products](https://user-images.githubusercontent.com/28401696/122072280-0d6fd500-ce15-11eb-9938-f4b546de58c0.png)

Reply:
Hi. pivot its a name of 3rd table relation when you want to load belongsToMany relation